adapter/analytics/analytics.go

```diff
@@ -510,7 +509,7 @@ func (m *manager) signedURL(subdir, filename string) (string, error) {
 	defer resp.Body.Close()
 
 	if resp.StatusCode != http.StatusOK {
-		return "", fmt.Errorf("status (code %d) returned from %s: %s", resp.StatusCode, m.analyticsURL, resp.Status)
+		return "", fmt.Errorf("status (code %d) returned from %s: %s", resp.StatusCode, u.Path, resp.Status)
```

Comment:
Can we use this u.String() instead of u.Path, so we have the whole thing? Thanks.